docs/topic/installer-actions.rst

```diff
@@ -21,6 +21,15 @@ JupyterHub is run from a python3 virtual environment located in ``/opt/tljh/hub`
 uses the system's installed python and is owned by root. It also contains a binary install 
 of `traefik <http://traefik.io/>`_. This virtual environment is completely managed by TLJH.
 
+.. note::
+	If you try to remove TLJH, revert this action using:
+
+	.. code-block:: bash
+
+		/opt/tljh/hub/bin/pip freeze | xargs sudo /opt/tljh/hub/bin/pip uninstall -y
```

Comment:
I'm curious what explicitly uninstalling stuff gives us over just removing the directory. The virtualenv should keep everything in there...

Reply:
I believe, nothing. I wrote this last night and I guess I wast just being extra paranoid and exaggerated :smile: